features/step_definitions/edit_page_steps.rb

```diff
@@ -9,7 +9,7 @@
 end
 
 Then(/^I should see the Draft Article$/) do
-  expect(JSON.parse(preview_page.text)['blocks'].find {|b| b["identifier"] == 'content'}['content']).to eq('test')
+  expect(JSON.parse(preview_page.text)['blocks'].find {|b| b["identifier"] == 'content'}['content']).to include('test')
```

Comment:
This assertion is too smart. I know that was already there. I would move this to a less smart assertion.